exercises/accumulate/.meta/hints.md

```diff
@@ -5,9 +5,12 @@ It is typical to call [#to_enum](http://ruby-doc.org/core-2.3.1/Object.html#meth
 Here is an additional test you could add:
 
 ```ruby
-def test_no_block_is_passed
+def test_accumulate_squares_when_no_block_is_passed
```

Comment:
That it squares is not really important.

"test working enumerator is returned when no block is passed" maybe?